influunt-app/app/views/controladores/wizard/dados-basicos.html

```diff
@@ -171,17 +171,16 @@
               </div>
 
               <div class="col-lg-12 col-sm-12 no-padding"
-                   data-ng-class="{ 'has-error': !dadosBasicos.alturaNumerica.$valid }">
+                   data-ng-class="{ 'has-error': errors.endereco.alturaNumericaNegativa }">
                 <div class="form-group">
                   <label class="control-label">{{ 'controladores.alturaNumerica' | translate }}:</label>
                   <input type="number"
                          name="alturaNumerica"
-                         min="0"
                          class="form-control"
                          data-ng-model="objeto.todosEnderecos[enderecoControladorIndex].alturaNumerica">
                   <p class="help-block error-msg"
-                     data-ng-if="!dadosBasicos.alturaNumerica.$valid">
-                     {{ 'controladores.validacaoAlturaNumerica' | translate }}
+                     data-ng-repeat="error in errors.endereco.alturaNumericaNegativa">
+                     {{ error }}
```

Comment:
Aquele erro de "não pode ficar em branco se localizacao2 não estiver preenchido" aparece aqui na altura numérica ou na localização 2? Se for aqui, ele não vai aparecer.